tests/unit/extraction_rules/test_audit_activity.py

```diff
@@ -332,6 +333,7 @@
         ATTRIBUTE_GCP_REGION: 'europe-north1',
         ATTRIBUTE_GCP_PROJECT_ID: 'dynatrace-gcp-extension',
         ATTRIBUTE_GCP_RESOURCE_TYPE: 'cloudsql_database',
+        ATTRIBUTE_GCP_INSTANCE_ID: 'dynatrace-gcp-extension:pawel-001-mysql',
```

Comment:
pointing to specific private pawel instance is not good idea I think. Better will be to create separate instance only for tests.

Reply:
it is just unit test string, not using any real instance

Reply:
Although I understand your reasoning @sq2gxo, I agree with @karolkieglerski on that, since Pawel or someone might remove this instance  because he wouldn't know that it's used for testing

Reply:
as we discussed today - it is just a string value but I've changed it anyway